pynicotine/gtkgui/widgets/treeview.py

```diff
@@ -529,15 +547,18 @@ def clear(self):
         self.widget.set_model(self.model)
 
     @staticmethod
-    def get_user_status_tooltip_text(icon_name):
+    def get_status_tooltip_text(icon_name):
 
-        if "away" in icon_name:
+        if icon_name == USER_STATUS_ICON_NAMES[UserStatus.AWAY]:
             return _("Away")
 
-        if "online" in icon_name:
+        if icon_name == USER_STATUS_ICON_NAMES[UserStatus.ONLINE]:
             return _("Online")
 
-        return _("Offline")
+        if icon_name == USER_STATUS_ICON_NAMES[UserStatus.OFFLINE]:
+            return _("Offline")
+
+        return icon_name
```

Comment:
There is never a case when we want the Tooltip Text to read the filename of the icon.

If the user status cannot be determined then "Offline" is the most sensible fallback to return.

Reply:
There was a conflict between the user status and transfer status columns, but I've resolved it in another way.